mmv1/products/apigee/KeystoreAliasesSelfsigned.yaml

```diff
@@ -0,0 +1,169 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+    name: 'KeystoreAliasesSelfSigned'
+    base_url: '{{keystore_id}}/aliases/{{alias}}'
+    create_url: '{{keystore_id}}/aliases?alias={{alias}}&format={{format}}'
+    self_link: '{{keystore_id}}/aliases/{{alias}}'
+    delete_url: '{{keystore_id}}/aliases/{{alias}}'
+    immutable: true
+    description: |
+      An Environment Keystore Alias for Self Signed Certificate Format in Apigee
+    references: !ruby/object:Api::Resource::ReferenceLinks
+      guides:
+        'Creating an environment': 'https://cloud.google.com/apigee/docs/api-platform/get-started/create-environment'
+      api: 'https://cloud.google.com/apigee/docs/reference/apis/apigee/rest/v1/organizations.environments.keystores/create'
```

Comment:
Could this be https://cloud.google.com/apigee/docs/reference/apis/apigee/rest/v1/organizations.environments.keystores.aliases/create instead?

Reply:
updated